daemons/gptp/common/avbts_message.hpp

```diff
@@ -125,7 +125,7 @@
 #define PTP_PTPTIMESCALE_BIT 3	/*!< PTPTIMESCAPE bit offset on PTPTIMESCALE byte*/
```

Comment:
This looks like this might be similar change - https://github.com/AVnu/OpenAvnu/pull/728/files

Reply:
Looks to be the right fix. Opened https://github.com/ni/Open-AVB/pull/42. Closing this.